src/Microsoft.OData.ModelBuilder/Vocabularies/VocabularyTermConfigurationExtensions.cs

```diff
@@ -46,13 +46,21 @@ internal static IEdmExpression ToEdmExpression(this string text)
         public static ChangeTrackingConfiguration HasChangeTracking(this FunctionConfiguration operationSource)
             => operationSource?.VocabularyTermConfigurations.GetOrCreateConfiguration<ChangeTrackingConfiguration>();
 
+        /// <summary>
+        /// <see cref="ChangeTrackingConfiguration"/> configuration
+        /// </summary>
+        /// <param name="operationSource">The <see cref="IEdmOperation"/> that can be built using <see cref="ODataModelBuilder"/>.</param>
+        /// <returns><see cref="ChangeTrackingConfiguration"/></returns>
+        public static ChangeTrackingConfiguration HasChangeTracking(this NavigationPropertyConfiguration operationSource)
+            => operationSource?.VocabularyTermConfigurations.GetOrCreateConfiguration<ChangeTrackingConfiguration>();
+
         /// <summary>
         /// <see cref="CountRestrictionsConfiguration"/> configuration
         /// </summary>
         /// <typeparam name="TEntity">The entity type of the navigation source.</typeparam>
         /// <param name="navigationSource">The <see cref="IEdmNavigationSource"/> that can be built using <see cref="ODataModelBuilder"/>.</param>
         /// <returns><see cref="CountRestrictionsConfiguration"/></returns>
-        public static CountRestrictionsConfiguration HasCountRestrictions<TEntity>(this EntitySetConfiguration<TEntity> navigationSource) where TEntity : class
+        public static CountRestrictionsConfiguration HasCountRestrictions<TEntity>(this NavigationSourceConfiguration<TEntity> navigationSource) where TEntity : class
```

Comment:
What caused this change?

Reply:
EntitySetConfiguration is derived from NavigationSourceConfiguration